src/runners/baseRunner/BaseRunnerResultAnalyzer.ts

```diff
@@ -41,35 +42,61 @@ export abstract class BaseRunnerResultAnalyzer {
 
     public feedBack(): ITestResult[] {
         const result: ITestResult[] = [];
+        const itemMap: Map<string, ITestItem> = new Map<string, ITestItem>();
         for (const test of this.tests) {
-            this.processTestItemRecursively(test, result);
+            this.flatTestItems(test, itemMap);
         }
+        this.parseResults(result, itemMap);
+        this.addSkippedTests(result, itemMap);
```

Comment:
The correctness of `parseResults` and `addSkippedTests` are strongly related to the execution order, as they share the state of itemMap.